example-playbooks/replace_node/replace_node.yml

```diff
@@ -26,6 +26,13 @@
 
     - name: Stop scylla-server on the node being replaced
       block:
+        - name: Set silence for the node that we are about to stop
+          shell: |
+            docker exec -it aalert amtool silence add instance="{{ replaced_node }}" --alertmanager.url="{{ alertmanager_url }}" --comment='Replacing node' --duration "{{ new_node_bootstrap_wait_time_sec|int * 2 }}s"
+          # We don't want the playbook to fail in case we don't succeed in silencing the alerts
```

Comment:
Why is that? What could be a good reason for such a failure?

Reply:
If monitoring is down or irresponsive for some reason, for example.